scripts/deploy.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Can you also turn this into gulp tasks. It's ok to move parts into a shell script if needed.